src/_shared/scss/_adverts-members.scss

```diff
@@ -0,0 +1,57 @@
+.adverts--memberss {
```

Comment:
`memberss` ?? Why the second s? 

Follow up question - Where do these classnames come from?

Reply:
Fair question - literally the class names on these are so annoying, and I don't know why the second s, this code has been through 3 different engineers in 2.5 years and it is weeeiiird. I had ambitions to make it my life mission to clean everything up and consolidate loads of the css, but sadly I wasn't allowed.

Reply:
Copying over some internal discussion to help any future git history explorers 😁 

The classes are inserted into the HTML of the template here:
https://github.com/guardian/commercial-templates/blob/master/src/manual-multiple/web/index.html

> the tone ('members' 'travel' 'live' etc.) comes from a DFP value, then for some reason there is an extra 's'
> Which means the class names are memberss, travels, lives
> It's a mystery to this day

😸 😸 Okay! Cheers for the explaination 👍

Reply:
Thanks Kate!